app/api/api_v1/sockets/web_sockets.py

```diff
@@ -51,3 +51,25 @@ async def websocket_video_endpoint(websocket: WebSocket) -> None:
     connection = WebSocketConnection(websocket, WebSocketTypeEnum.VIDEO)
     await socket_connection_manager.connect(connection)
     await socket_connection_manager.relay_video_frames(connection)
+
+
+@router.websocket("/ws/webrtc/controller")
+async def websocket_webrtc_controller_endpoint(websocket: WebSocket) -> None:
+    socket_connection_manager = SocketConnectionManager()
+    connection = WebSocketConnection(websocket, WebSocketTypeEnum.WEBRTC_CONTROLLER)
+    try:
+        if await socket_connection_manager.connect(connection):
+            await socket_connection_manager.start_webrtc_signaling(connection)
+    except Exception:
+        socket_connection_manager.disconnect(connection)
```

Comment:
Similar to the above, would be good to expand the try/catch with something like:

```suggestion
    except websockets.exceptions.ConnectionClosedError as e:
        socket_connection_manager.disconnect(connection)
        logger.error(f"WebSocket connection closed with an error: {e.code} - {e.reason}")
    except Exception as e:
        logger.error(f"An unexpected error occurred during WebSocket communication: {e}")
```

**PS:** Same question to the `webrtc/peer` below.